.pre-commit-config.yaml

```diff
@@ -39,7 +39,7 @@ repos:
         files: >-
           (
           ?^ci/docker/conda-python-emscripten\.dockerfile$|
-          ?^ci/docker/python-wheel-windows-test-vs2019\.dockerfile$|
+          ?^ci/docker/python-.*-wheel-windows-test-vs2019.*\.dockerfile$|
```

Comment:
Does this match `python-wheel-windows-vs2019-base.dockerfile`?

Reply:
Probably not, but this is asking for more and more things that are outside of the scope of this PR...